requirements.txt

```diff
@@ -1,2 +1,3 @@
 git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/XBlock.git@cdyer/scorable-mixin
```

Comment:
Yes, this is just temporary while some work @jcdyer is doing is still in flight. I want this dependency pulled in for the sake of running tests but I definitely would not merge this file!